Website/src/rendering/src/assets/css/components/Navigation/ShopNavigation.css

```diff
@@ -3,13 +3,11 @@
   @apply lg:h-20;
```

Comment:
The content of the shop navigation is not centered (top/bottom) in its container. It is ok to use pixels for the height in all breakpoints like we do in `MainNavigation.css`.